integration-test/src/test/java/org/sagebionetworks/warehouse/WarehouseTestHelper.java

```diff
@@ -3,6 +3,12 @@
 import java.time.Instant;
 
 public interface WarehouseTestHelper {
+	
+	public static final int WAREHOUSE_QUERY_EXPIRATION_HOURS = 2;
+	
+	default void assertWarehouseQuery(String queryString) throws Exception {
+		assertWarehouseQuery(queryString, WAREHOUSE_QUERY_EXPIRATION_HOURS);
```

Comment:
there is an assumption on the index in the stack trace.
let's get rid of the call to take hours.